conan/tools/google/bazeldeps.py

```diff
@@ -32,7 +33,8 @@ def generate(self):
         self._save_main_buildfiles(content)
 
     def _save_dependendy_buildfile(self, dependency, buildfile_content):
-        filename = 'conandeps/{}/BUILD'.format(dependency.ref.name)
+        filename = '{}/conandeps/{}/BUILD'.format(self._conanfile.install_folder, dependency.ref.name)
```

Comment:
If we can avoid having this path absolute to ``install_folder``, better, because the generation should already happen in the right location.

Reply:
If I run `conan install . -if .conan` then currently it hardcodes the path to 'conandeps' but it is actually written to '.conan/conandeps'.

I assumed the absolute path was fine since the path to the package is already absolute.

Reply:
It might be fine atm, but I am thinking about the path to migrate to 2.0, in which the ``install_folder`` dissapears. But it is fine at the moment, I think this can be merged now and we'll check when we finally remove ``install_folder``

Reply:
Understood. I just wanted to patch this up and get something that works for me with Conan as it is today. 
I haven’t used Conan 2 yet.